Triangulation_2/include/CGAL/Triangulation_2_projection_traits_3.h

```diff
@@ -16,12 +16,16 @@
 
 #include <CGAL/license/Triangulation_2.h>
 
+#include <CGAL/disable_warnings.h>
```

Comment:
Why? And as far as I see you don't ever re-enable warnings.

+ Needs CGAL_NO_DEPRECATED_CODE